library/tasks/ModifyGtf.wdl

```diff
@@ -0,0 +1,51 @@
+task ReplaceGeneNameWithGeneID {
+  File original_gtf
+
+  #runtime values
+  String docker = "quay.io/humancellatlas/modify-gtf:0.1.0"
+  Int machine_mem_mb = 8250
+  Int cpu = 1
+  Int disk = ceil(size(original_gtf, "Gi") * 2) + 1
+  Int preemptible = 3
+  String modified_gtf_location = "gene_id_as_gene_name.gtf"
+
+  meta {
+    description: "Modifies the gene_name field in a gtf to contain the values of gene_id instead."
+  }
+
+  parameter_meta {
+    original_gtf: "The gtf to modify."
+    docker: "(optional) the docker image containing the runtime environment for this task"
+    modified_gtf_location: "(optional) the name to save the modified gtf file under"
+    machine_mem_mb: "(optional) the amount of memory (MiB) to provision for this task"
+    cpu: "(optional) the number of cpus to provision for this task"
+    disk: "(optional) the amount of disk space (GiB) to provision for this task"
+    preemptible: "(optional) if non-zero, request a pre-emptible instance and allow for this number of preemptions before running the task on a non preemptible machine"
+  }
+
+  command {
+    set -e
+
+    if file --mime-type "${original_gtf}" | grep  gzip; then
+      gunzip -c  "${original_gtf}" > input.gtf
```

Comment:
Why are we supporting both zipped and non-zipped gtf files? Can we choose one to support and stick with it?

Reply:
I just spoke to Hornet and it seems like they use externally-provided gtf files, and there isn't always certainty whether or not they come compressed or not, hence the logic to unzip if necessary.

Reply:
Shouldn't the adapter workflows take care of providing the workflow consistent data? Lets not let it get in the way of this PR, but its something we should keep in mind.

Reply:
Note that we also develop these pipelines for users, who may not be as consistent. But I agree, it's reasonable to set standards or define inputs as long as we communicate the pipeline's expectations clearly in our documentation.